db/src/db.rs

```diff
@@ -390,6 +393,24 @@ pub fn read_db(conn: &rusqlite::Connection) -> Result<DB> {
     Ok(DB::new(partition_map, schema))
 }
 
+use itertools;
```

Comment:
I moved the `type` definitions to types and lifted the `use` statements to the top of the file.  These were just oversights.